### PR TITLE
feature/databricks-compatibility

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

Databricks compatibility 🧱

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide an explanation as to how the change is non-breaking below.)

Same as the source, since there was a breaking change only a while ago it would not make sense to do a breaking change here as well.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature #42 
- [ ] No 

**How did you test the PR changes?** 

 

- [X] CircleCi  
- [X] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [X] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🧱

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 packages:
- - package: fivetran/stripe_source
-   version: [">=0.7.0", "<0.8.0"] 
+#  - package: fivetran/stripe_source
+#    version: [">=0.7.0", "<0.8.0"] 
+
+  - git: https://github.com/fivetran/dbt_stripe_source.git
+    revision: feature/databricks-compatibility
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:7]: To be switched before release.

**Review by fivetran-sheringuyen — APPROVED** [review]
Hey @fivetran-joemarkiewicz, everything looks good to me. I just had the same question I had in the source - and a reminder to update `packages.yml` before release as you've already commented for yourself :) 

Do you intend to keep those `stripe_*` schema names for `integration_tests/ci/sample.profiles.yml`?

**Comment by fivetran-joemarkiewicz** [comment]
> Hey @fivetran-joemarkiewicz, everything looks good to me. I just had the same question I had in the source - and a reminder to update `packages.yml` before release as you've already commented for yourself :)
> 
> Do you intend to keep those `stripe_*` schema names for `integration_tests/ci/sample.profiles.yml`?

Thanks @fivetran-sheringuyen great call out, just updating those now!

File: packages.yml (hunk 1 above)
Comment (fivetran-joemarkiewicz) [inline packages.yml:7]:
```suggestion
  - package: fivetran/stripe_source
    version: [">=0.7.0", "<0.8.0"] 

```

**Pull request merged**